### ISSUE TITLE
Import profiling results of command with ',' comma fails

### ISSUE THREAD

**Issue by aosewski** [AMDResearch/omniperf#76]
I've successfully run full (all IP blocks) profiling of my program, which command line contains multiple commas. However when I try to import collected data to Omniperf database I get following error:

```
Found sysinfo file                                                                                                                                                                                                 
Traceback (most recent call last):                                                                                                                                                                                 
  File "/opt/omniperf/1.0.6/bin/omniperf", line 663, in <module>                                                                                                                                                   
    main()                                                                                                                                                                                                         
  File "/opt/omniperf/1.0.6/bin/omniperf", line 609, in main                                             
    mongo_import(args, False)                                                                                                                                                                                      
  File "/opt/omniperf/1.0.6/bin/omniperf", line 205, in mongo_import                                     
    connectionInfo, Extractionlvl = csv_converter.parse(args, profileAndImport)                                                                                                                                    
  File "/opt/omniperf/1.0.6/bin/utils/csv_converter.py", line 145, in parse                              
    db = "omniperf_" + args.team + "_" + name + "_" + soc                                                                                                                                                          
TypeError: can only concatenate str (not "numpy.int64") to str
```
And this is because my `sysinfo.csv` file contains my program command which contains multiple commas and this breaks `csv` file structure. 

I'd be grateful for fixing this bug or any hits how to overcome this.

**Comment by coleramos425** [comment]
A program name containing commas is certainly an edge case that would break our import process. We'll get this patched in v1.0.7

However, looking at your included traceback this seems to be occurring because of the `-t/--team` and/or `-n/--name` you passed. They were likely integers we failed to convert to str. This is a separate issue, but one we'll also patch in v1.0.7

**Comment by coleramos425** [comment]
Issues have been fixed. This is will be included in our release early next week. Thank you.
